### ISSUE TITLE
Option to save comicinfo/metadata and poster on the manga root folder.

### ISSUE THREAD

**Issue by MrChuw** [mihonapp/mihon#652]
### Describe your suggested feature

Save the metadata and the poster in the manga's root folder, so that it is easier to import it to other services.

Source
├── Manga Name
│   ├── Chapter 1.cbz
│   ├── Chapter 2.cbz
│   ├── Chapter 3.cbz
│   ├── Poster.jpg
│   └── ComicInfo.xml

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by HamusutaW** [comment]
Totally agree! This allows for better portability of downloaded comics.

Especially going through the migration from Tachiyomi to Mihon. I have a large number of manga downloaded in Tachiyomi that are not recognized in Mihon. Although it can be read, the icon shows that it has not been downloaded, and there is no response when clicking the download icon.

A similar scenario also occurred when I changed to a new phone and I had to re-download all comics to ensure that the download icon was displayed correctly.

**Comment by MajorTanya** [comment]
> Totally agree! This allows for better portability of downloaded comics.
> 
> Especially going through the migration from Tachiyomi to Mihon. I have a large number of manga downloaded in Tachiyomi that are not recognized in Mihon. Although it can be read, the icon shows that it has not been downloaded, and there is no response when clicking the download icon.
> 
> A similar scenario also occurred when I changed to a new phone and I had to re-download all comics to ensure that the download icon was displayed correctly.

That wouldn't have been necessary, you could have just invalidated the download index from the advanced settings.

**Comment by XI-ryimi** [comment]
很需要这个，我有大量来自Manga Manager标注的漫画，mihon暂时还无法识别元数据分组，软件目前的本地功能还是有些羸弱

**Comment by Comprehensive-Jason** [comment]
I agree, this would be a great feature to easily convert non-local series to local series.
